content/news/09-02-21-the-open-revolution-talk

```diff
@@ -0,0 +1,20 @@
+title: "The Open Revolution: Making a radically fairer and free future"
```

Comment:
I'd tweak title a bit to mention what this is i.e. a talk as this is a blog post *about the talk* e.g.

"Talk at Open Access Week 2021: An Open Revolution for a Fairer and Freer Future"